db.go

```diff
@@ -52,11 +52,44 @@ func (net *Network) GetName() string {
 	return net.Addr
 }
 
+type MessageFilter int
+
+const (
+	FilterNone      MessageFilter = 0
+	FilterHighlight               = 1
```

Comment:
If you use explicit values, you also need to specify explicitly the type. Otherwise the type of `FilterHighlight` and `FilterMessage` will be `int` instead of `MessageFilter`.

But here I think `iota` could be used instead.